internal/naming/labels.go

```diff
@@ -40,6 +40,10 @@ const (
 	// LabelMovePGWalDir is used to identify the Job that moves an existing pg_wal directory.
 	LabelMovePGWalDir = labelPrefix + "move-pgwal-dir"
 
+	// LabelCollectorDiscovery is added to Pods running the OpenTelemetry "collector"
+	// container to support discovery by Prometheus
+	LabelCollectorDiscovery = labelPrefix + "crunchy-otel-collector"
```

Comment:
If we are planning on not changing the Prometheus setup, then I think the pods we want Prometheus to scrape should have the usual metrics labels? Ugh, but that label is `crunchy-postgres-exporter`, which I do not want on these pods, so we should adjust the prometheus config to look for either, I suppose. I'm looking at the monitoring stack, so I'll add that to my ticket.

Reply:
Just adding this note after our discussion: in prometheus, we'll add another scrape job to get the OTel collector; and we'll make sure to _only_ add the port (and the label?) to those pods that have metrics exports on